### PR TITLE
Bh/output updates

### PR DESCRIPTION
**Pull request by bheni**

### CODE REVIEW

File: go/libraries/doltcore/mvdata/data_mover.go [hunk 1]
```diff
@@ -169,17 +170,19 @@ func NewDataMover(ctx context.Context, root *doltdb.RootValue, fs filesys.Filesy
 	return imp, nil
 }
 
-func (imp *DataMover) Move(ctx context.Context) error {
+func (imp *DataMover) Move(ctx context.Context) (int64, error) {
```
Comment (zachmu) [inline go/libraries/doltcore/mvdata/data_mover.go]: You should document the new return param

File: go/libraries/doltcore/remotestorage/chunk_store.go [hunk 2]
```diff
@@ -230,8 +230,8 @@ func (dcs *DoltChunkStore) GetManyCompressed(ctx context.Context, hashes hash.Ha
 }
 
 const (
-	getLocsBatchSize      = 4096
-	getLocsMaxConcurrency = 32
+	getLocsBatchSize      = 128 * 1024
```
Comment (zachmu) [inline go/libraries/doltcore/remotestorage/chunk_store.go]: This file unrelated?

Comment (bheni) [reply go/libraries/doltcore/remotestorage/chunk_store.go]: yeah.  Pullin a sneaky.

File: go/libraries/doltcore/mvdata/data_mover.go (hunk 1 above)
Comment (bheni) [reply go/libraries/doltcore/mvdata/data_mover.go]: done

**Pull request merged**